plugins/field-grid-dropdown/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+    "compilerOptions": {
+        "baseUrl": "./",
+        "outDir": "dist",
+        "declaration": true,
+        "declarationMap": true,
+        "module": "ES2015",
+        "moduleResolution": "node",
+        "target": "ES2015",
+        "strict": true,
+    },
+    // NOTE: `test/**/*` is automatically included in `blockly-scripts start`.
+    // Only src matters for production builds.
+    "exclude": ["test/**/*", "dist/**/*", "build/**/*"]
```

Comment:
Similar question about excludes as in previous PR: https://github.com/google/blockly-samples/pull/1651#discussion_r1154982424

Reply:
Responded in the previous PR!

Tldr; this is a mostly innocuous change that just makes VS Code complain less when the TS Config file is open with artifacts built. It doesn't actually impact the pipeline build since those artifacts aren't included in git anyway.